templates/blocks/columns_tasks.twig

```diff
@@ -28,71 +28,42 @@
-					</div>
-					<div class="step-info">
-						<div class="row">
-							{% for col in fields.columns %}
-								<div class="col" data-id="{{ loop.index }}">
-									<h5
-										data-ga-category="Columns Block"
-										data-ga-action="Title"
-										data-ga-label="n/a">
-											{{ col.title }}
-									</h5>
-								</div>
-							{% endfor %}
-						</div>
-						<div class="row">
-							{% for col in fields.columns %}
-								<div class="col" data-id="{{ loop.index }}">
-									{% if ( col.description ) %}
-										<p {% if ( col.cta_text and col.cta_link ) %}class="has-cta"{% endif %}>
-											{{ col.description|e('wp_kses_post')|raw }}
-										</p>
+								<h5 data-ga-category="Columns Block"
+									data-ga-action="Title"
+									data-ga-label="n/a">
+										{{ col.title }}
+								</h5>
+								{% if ( col.description ) %}
+									<p {% if ( col.cta_text and col.cta_link ) %}class="has-cta"{% endif %}>
+										{{ col.description|e('wp_kses_post')|raw }}
+									</p>
+								{% endif %}
+								{% if ( col.attachment ) %}
+									{% if ( col.cta_link ) %}
+										<a
+											href="{{ col.cta_link|e('esc_url') }}"
+											data-ga-category="Columns Block"
+											data-ga-action="Image"
+											data-ga-label="{{ col.cta_link|e('esc_url') }}">
 									{% endif %}
-								</div>
-							{% endfor %}
-						</div>
-						<div class="steps-action">
-							<div class="row">
-								{% for col in fields.columns %}
-									<div class="col" data-id="{{ loop.index }}">
-										{% if ( col.attachment ) %}
-												{% if ( col.cta_link ) %}
-													<a
-														href="{{ col.cta_link|e('esc_url') }}"
-														data-ga-category="Columns Block"
-														data-ga-action="Image"
-														data-ga-label="{{ col.cta_link|e('esc_url') }}">
-												{% endif %}
-														<img src="{{ col.attachment }}" alt=""/>
-												{% if ( col.cta_link ) %}
-													</a>
-												{% endif %}
-										{% endif %}
-									</div>
-								{% endfor %}
-							</div>
-							<div class="row">
-								{% for col in fields.columns %}
-									<div class="col" data-id="{{ loop.index }}">
-										{% if ( col.cta_text and col.cta_link ) %}
-											<a
-												class="btn btn-small btn-medium {{ is_campaign ? 'btn-primary' : 'btn-secondary' }}"
-												href="{{ col.cta_link|e('esc_url') }}"
-												{{  col.link_new_tab  ? 'target="_blank"' }}
-												data-ga-category="Columns Block"
-												data-ga-action="Call to Action"
-												data-ga-label="{{ col.cta_link|e('esc_url') }}">
-													{{ col.cta_text }}
-											</a>
-										{% endif %}
-									</div>
-								{% endfor %}
+										<img src="{{ col.attachment }}" alt=""/>
+									{% if ( col.cta_link ) %}
+										</a>
+									{% endif %}
+								{% endif %}
+								{% if ( col.cta_text and col.cta_link ) %}
+									<a
```

Comment:
We need to add some margin above this one now (or below the element above it). Previously this margin was ensure by the default `p` margin-bottom, but with flexbox that margin is not used.

Reply:
Actually I saw this wrong. The margin is there, but in the child theme there is a rule to set the p bottom margin to 0. Then they had a rule using `.steps-action` to set a top margin to the CTA, which is now not applying anymore.

Reply:
[From their commit message](https://github.com/greenpeace/planet4-child-theme-netherlands/commit/32c803d93c8b27b0fab457ed41cab72083f1b194#diff-3f958b82f02ca6ad0d9a567fa9ebc9c6R77) it looks like that rule was only added to provide alignment. So maybe a lot of their custom columns css can be removed if we fixed those things in base. @dirkjf can you confirm/correct my assumptions?

Reply:
@Inwerpsel, yes, the rule was added simply for alignment.

Reply:
@dirkjf So if we merge this PR we would need to ensure [this rule](https://github.com/greenpeace/planet4-child-theme-netherlands/blob/develop/assets/css/scss/blocks/_columns.scss#L116) is removed before we deploy it (maybe best to also double check if more things can be removed or rules need to be adapted to the new markup). We need to make sure both changes are in the same deploy. Would that work for you?

Reply:
Let's unblock this. I can pause the GPNL deploy on next release and send a notification for this rule to be removed before approving production.